packages/ckeditor5-watchdog/src/contextwatchdog.ts

```diff
@@ -194,10 +197,15 @@ export default class ContextWatchdog<TContext extends Context = Context> extends
 	 * @param itemId The item ID.
 	 * @returns The item instance or `undefined` if an item with a given ID has not been found.
 	 */
-	public getItem( itemId: string ): unknown {
-		const watchdog = this._getWatchdog( itemId );
+	public getItem( itemId: string ): unknown | undefined {
+		try {
+			const watchdog = this._getWatchdog( itemId );
 
-		return watchdog._item;
+			return watchdog._item;
+		}
+		catch {
+			return undefined;
+		}
```

Comment:
I'm not sure if this change is valid, but docs say it should return `undefined` if not found but it was throwing error instead. If it's valid change, then tests should be adjusted.